optax/_src/loss.py

```diff
@@ -352,9 +371,28 @@ def loop_body(prev, x):
 
   # last row needs to be updated with the last epsilon transition
   logalpha_phi_last = update_phi_score(logalpha_phi[-1], logalpha_emit[-1])
+  logalpha_phi = logalpha_phi.at[-1].set(logalpha_phi_last)
 
   # extract per_seq_loss
   one_hot = jax.nn.one_hot(labellens, num_classes=maxlabellen + 1)  # [B, N+1]
   per_seq_loss = -jnp.einsum('bn,bn->b', logalpha_phi_last, one_hot)
 
-  return per_seq_loss
+  return per_seq_loss, logalpha_phi, logalpha_emit
+
+
+def ctc_loss(logits: chex.Array,
+             logitpaddings: chex.Array,
+             labels: chex.Array,
+             labelpaddings: chex.Array,
+             blank_id: int = 0,
+             logepsilon: float = -1e5) -> chex.Array:
+  """Computes CTC loss.
+
+  See docstring for `ctc_loss_with_forward_score` details.
+
+  Returns:
+    (B,)-array containing loss values for each sequence in the batch.
+  """
+  return ctc_loss_with_forward_probs(
+      logits, logitpaddings, labels, labelpaddings,
+      blank_id=blank_id, logepsilon=logepsilon)[0]
```

Comment:
Could you change it to
```
per_seq_loss, _, _ = ctc_loss_with_forward_probs(...)
return per_seq_loss
```
for better readability? (indexing is easy to overlook)

Reply:
Thanks for the review. Done.